Dockerfile

```diff
@@ -0,0 +1,19 @@
+# Dockerfile
+FROM php:8.3.2-cli
+
+RUN apt-get update -y \
+  && apt-get install -y libmcrypt-dev \
+  && apt-get clean \
+	&& docker-php-ext-install bcmath \
+  && rm -rf /var/lib/apt/lists/* /var/cache/apt/archives/*
+
+COPY --from=composer:latest /usr/bin/composer /usr/bin/composer
+
+WORKDIR /app
+COPY . /app
+
+RUN composer install
+RUN php artisan key:generate
```

Comment:
This runs when the image is build. It currently does not add any value here. We should add a .dockerignore file so that for example .envs and secrets are not in the docker image.

Reply:
There is a dockerignore file, I'll add the secrets folder. I think we don't need to exclude the .env file.